### PR TITLE
Added GitHub action workflow for greeting contributors

### LINKED ISSUE #224: [Feature] GitHub action workflow for greeting contributors
### Description

Description
The current project lacks an automated action to greet contributors. GitHub Actions can significantly enhance project automation. I propose adding a GitHub Action workflow to greet new pull request (PR) and issue creators, fostering a welcoming environment.

Proposed Changes:
Implement a GitHub Action workflow that triggers on new PRs and issues. The workflow should include a greeting message to acknowledge and appreciate the contributors for their efforts.

### Screenshots

_No response_

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Click-The-Edible-Game/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [ ] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Click-The-Edible-Game/blob/main/.github/CODE_OF_CONDUCT.md)".

### PR DESCRIPTION
**Pull request by praptisharma28**
# Fixes Issue🛠️

Closes #224

# Description👨‍💻 

Added 4 files

# Type of change📄

- [] New feature (non-breaking change which adds functionality)

# How this has been tested✅

on github actions

# Checklist✅ 

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have added demonstration in the form of GIF/video file
- [] I am an Open Source Contributor

### CODE REVIEW

**Comment by praptisharma28** [comment]
@Rakesh9100 add the tags as per the files I have added on issue #224, thanks:)

**Comment by Rakesh9100** [comment]
No other workflows like checking title, codes is needed. Only add the workflow to greet contributors @praptisharma28

**Comment by praptisharma28** [comment]
done @Rakesh9100

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  greeting:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Hey, @${{ github.actor }} welcome to URLShortener repository.🎊 Thank you so much for taking the time to point this out.🙌'
```
Comment (Rakesh9100) [inline .github/workflows/greetings.yml:15]: Why URLShortener repo is mentioned @praptisharma28

**Comment by praptisharma28** [comment]
done @Rakesh9100

**Comment by Rakesh9100** [comment]
Again codeql and other unnecessary files are added, do fix @praptisharma28

**Comment by Rakesh9100** [comment]
Merged 🎉 @praptisharma28

**Pull request merged**